wallet/service/openapi.json

```diff
@@ -31,7 +31,7 @@
           {
             "in": "header",
             "name": "Authorization",
-            "description": "The token used to access the protected methods. The format is `VWT <TOKEN>`. VWT is a non-standard scheme that stands for Vega Wallet Token. If this header is missing when accessing protected methods, the request will be rejected as an unauthorized request (401 Unauthorized).",
+            "description": "The token used to access the protected methods. The format is `VWT YOUR_TOKEN`. VWT is a non-standard scheme that stands for Vega Wallet Token. If this header is missing when accessing protected methods, the request will be rejected as an unauthorized request (401 Unauthorized).",
```

Comment:
non-blocker, and very nit-picky, but given the format is in all-caps, I think it makes more sense to have some way of indicating that the `YOUR_TOKEN` bit is a placeholder for, well, your token... I'm not sure why the `<>` characters were removed, but if they caused problems, the specs mention CommonMark syntax can be used in the description field, so `\\<YOUR_TOKEN\\>` could be used?

Reply:
I removed them because the formatting breaks en route to the docs so it inconsistently renders as &lt; - but if the slashes are likely to work, we can try them instead

Reply:
addressed